modules/log/event_writer.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package log
+
+import (
+	"fmt"
+)
+
+type EventWriter interface {
+	EventWriterBase
```

Comment:
There is no extra methods added, why we need the two interfaces?

Reply:
This is a framework consideration.

Although they share the same methods at the moment, they are not the same thing.

In the future, EventWriter may has its own interface methods to force developers to implement something.

And I added more comments in code 1bb5e96e6abbdd3cc920add7ddc822e7777bdfa5